shared/js/background/dnr-service-worker-initiated.js

```diff
@@ -20,7 +20,17 @@ import {
 export async function ensureServiceWorkerInitiatedRequestExceptions (config) {
     const removeRuleIds = [SERVICE_WORKER_INITIATED_ALLOWING_RULE_ID]
     const addRules = []
-    if (config.features.serviceworkerInitiatedRequests?.exceptions?.length) {
+
+    if (config.features.serviceworkerInitiatedRequests?.state !== 'enabled') {
+        // All ServiceWorker initiated request blocking is disabled.
+        addRules.push(generateDNRRule({
```

Comment:
I kept the diff simple by duplicating the rule details. I figured that was best, since this code is pretty tiny and can go away in the future anyway.